projects/canopy/src/lib/filter-container/filter-container.component.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  ElementRef,
+  forwardRef,
+  HostBinding,
+  HostListener,
+  OnDestroy,
+  Renderer2,
+  ViewEncapsulation,
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { keyName } from '../utils/keyboard-keys';
+import { LgButtonToggleDirective } from '../button/button-toggle/button-toggle.directive';
+
+import { LgFilterContainerPanelComponent } from './filter-container-panel/filter-container-panel.component';
+import {
+  lgFilterContainerPanelIdPrefix,
+  lgFilterContainerToggleIdPrefix,
+} from './filter-container.constants';
+
+let nextUniqueId = 0;
+
+@Component({
+  selector: 'lg-filter-container',
+  templateUrl: './filter-container.component.html',
+  styleUrls: [ 'filter-container.component.scss' ],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    class: 'lg-filter-container',
+  },
+})
+export class LgFilterContainerComponent implements AfterContentInit, OnDestroy {
+  private uniqueId = nextUniqueId++;
+  private subscription: Subscription;
+
+  @HostBinding('id') id = `lg-filter-container-${this.uniqueId}`;
+
+  @ContentChild(forwardRef(() => LgButtonToggleDirective))
+  filterContainerToggle: LgButtonToggleDirective;
+  @ContentChild(forwardRef(() => LgFilterContainerPanelComponent))
```

Comment:
Maybe a new line between the two `ContentChild`s?

Reply:
I can but I tend to group them together 😅

Reply:
Done